apps/backend/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "oko-wallet-backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "ts-node-dev --files --quiet src/index.ts",
+    "start-prod": "node dist/index.js",
+    "backend": "node dist/index.js",
+    "backend:dev": "ts-node-dev --files --quiet src/index.ts",
+    "build": "tsc --skipLibCheck",
+    "ts": "tsc --pretty",
+    "lint": "eslint ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/madfish-solutions/oko-wallet-backend.git"
+  },
+  "author": "Madfish",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/madfish-solutions/oko-wallet-backend/issues"
+  },
+  "homepage": "https://github.com/madfish-solutions/oko-wallet-backend#readme",
+  "dependencies": {
+    "axios": "^1.2.1",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.15",
+    "@types/node": "^18.11.15",
+    "@typescript-eslint/eslint-plugin": "^5.46.1",
+    "@typescript-eslint/parser": "^5.46.1",
+    "eslint": "^8.29.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "prettier": "^2.8.1",
```

Comment:
seems like we don't need this deps, cuz `eslint` is configured in the root folder